src/components/Pagination.vue

```diff
@@ -10,7 +10,7 @@
         <slot name="previous">&lt;</slot>
       </a>
     </li>
-    <li v-for="(item, key) in pages" :key="key" :class="[bem('item'), page === item ? bem('item', 'active') : '']" >
+    <li v-for="item in pages" :key="item" :class="[bem('item'), page === item ? bem('item', 'active') : '']" >
```

Comment:
page is already used, but it should probably be renamed to currentPage.
this would be a breaking change but do-able.